issue_15/script.js

```diff
@@ -0,0 +1,87 @@
+const showBtn = document.getElementById('js-show-btn');
+const modal = document.getElementById('js-modal');
+const closeBtn = document.getElementById('js-close-btn');
+const submitBtn = document.getElementById('js-submit-btn');
+
+
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function renderList(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.href = item.a;
+        a.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+async function fetchData() {
+    try {
+        const URL = 'https://api.json-generator.com/templates/NluaaELSLhVe/data?access_token=zjrk7bmqgvptk4dcs8p8bhs5n9sk7m1ecs65gvq4';
```

Comment:
```suggestion
        const endpoint = 'https://api.json-generator.com/templates/NluaaELSLhVe/data?access_token=zjrk7bmqgvptk4dcs8p8bhs5n9sk7m1ecs65gvq4';
```
I share this because it has been pointed out to me before.

Reply:
A new drawer has been added to the variable name. Thank you very much:v:
f73b6862bcb522d2ddf1bfcd7b056535847be49e